tests/test_wave_01.py

```diff
@@ -122,8 +122,9 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
In this case where the code is checking individual keys of a dictionary, I recommend including all of the relevant keys.  
```suggestion
    watched_movie = updated_data["watched"][0]
    assert watched_movie["title] == MOVIE_TITLE_1
    assert watched_movie["rating"] == RATING_1
    assert watched_movie["genre"] == GENRE_1
```

Reply:
Ahh, that makes sense--testing for all relevant keys makes for a more rigorous test and (maybe?) also yields greater code coverage.